website/docs/docs/building-a-dbt-project/building-models/python-models.md

```diff
@@ -0,0 +1,560 @@
+---
+title: "Python models"
+---
+
+:::info Beta
+
+This feature is arriving in dbt Core v1.3, which is currently in beta. Note that only [specific data warehouses](#specific-data-warehouses) support dbt Python models, and that support for some dbt Cloud capabilities will be in progress during the beta period.
+
+We encourage you to:
+- [Read the original discussion](https://github.com/dbt-labs/dbt-core/discussions/5261)
+- [Weigh in on our developing best practices](https://github.com/dbt-labs/docs.getdbt.com/discussions/1811)
+- Join the **#beta-feedback-python-models** channel in the [dbt Community Slack](https://www.getdbt.com/community/join-the-community/)
+
+Below, you'll see sections entitled "❓ **Our questions**." We're working to develop our opinionated recommendations ahead of the final release this October—and you can help! Comment in the GitHub discussions, leave thoughts in Slack, talk about it with colleagues and friends.
+:::
+
+## What is a Python model in dbt?
+
+A dbt Python model is a function that reads in dbt sources or models, applies a series of transformations, and returns a transformed dataset. Just like SQL models!
```

Comment:
It would be nice to provide a tl;dr for the linked discussion/issue so we can spark interest for people who don't like to go off and read more. Can you add just another sentence or two? Possibly an example? I can help if you need it, @jtcohen6, but I wasn't sure what exactly to say. 
```suggestion
dbt Python models will help you solve use cases that you can't solve with SQL, and bring you closer to dbt when the data warehouse is your principal tool. 

A dbt Python model is a function that reads in dbt sources or models, applies a series of transformations, and returns a transformed dataset. Just like SQL models!
```